llvm/tools/llvm-exegesis/lib/SnippetRepetitor.cpp

```diff
@@ -31,7 +31,11 @@ class DuplicateSnippetRepetitor : public SnippetRepetitor {
       if (!Instructions.empty()) {
         // Add the whole snippet at least once.
```

Comment:
We no longer need the special case to add the snippet once, because we're now always adding the snippet in an all-or-nothing way.